code/modules/research/rdmachines.dm

```diff
@@ -48,20 +49,28 @@ var/global/list/rnd_machines = list()
 	src.disable_wire = pick(w)
 	w -= src.disable_wire
 
+	if (ticker)

```

Comment:
so to make it initialize you have to decon and rebuild it? all this stuff's new is called before the ticker is made